src/shared/components/search-filters/search-filters.component.ts

```diff
@@ -0,0 +1,177 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { SearchInput } from './domain/search-input.model';
+import { SearchService, WindowService, OrderService } from '../../services';
+import { Jurisdiction, CaseTypeLite, CaseState } from '../../domain';
+import { JurisdictionService } from '../../services/jurisdiction';
+
+const JURISDICTION_LOC_STORAGE = 'search-jurisdiction';
+const META_FIELDS_LOC_STORAGE = 'search-metadata-fields';
+const FORM_GROUP_VALUE_LOC_STORAGE = 'search-form-group-value';
+const CASE_TYPE_LOC_STORAGE = 'search-caseType';
+@Component({
+  selector: 'ccd-search-filters',
+  templateUrl: './search-filters.html',
+})
+
+export class SearchFiltersComponent implements OnInit {
+
+  @Input()
+  jurisdictions: Jurisdiction[];
+
+  @Input()
+  autoApply: boolean;
+
+  @Output()
+  onApply: EventEmitter<any> = new EventEmitter();
+
+  @Output()
+  onReset: EventEmitter<any> = new EventEmitter();
+
+  searchInputs: SearchInput[];
+  searchInputsReady: boolean;
+
+  selected: {
+    jurisdiction?: Jurisdiction,
+    caseType?: CaseTypeLite,
+    formGroup?: FormGroup,
+    caseState?: CaseState,
+    page?: number,
+    metadataFields?: string[]
+  };
+
+  selectedJurisdictionCaseTypes?: CaseTypeLite[];
+
+  formGroup: FormGroup = new FormGroup({});
+
+  constructor(private searchService: SearchService,
+    private orderService: OrderService,
+    private jurisdictionService: JurisdictionService,
+    private windowService: WindowService) {
+  }
+
+  ngOnInit(): void {
+    this.selected = {};
+    const jurisdiction = this.windowService.getLocalStorage(JURISDICTION_LOC_STORAGE);
```

Comment:
Extracting Search component into CaseUIToolKit also includes LocalStorage persistent functionality, so other parties other than case management using this component may get this feature even if they haven't asked for or even if they don't want it. But I guess the business might be able to answer it more appropriately.

Let me know your thoughts

Reply:
Good point and as discussed this will be more like a feature of the existing extracted component to third parties so it is ok.